upb/def.c

```diff
@@ -100,6 +100,8 @@ struct upb_msgdef {
   /* Is this a map-entry message? */
   bool map_entry;
   upb_wellknowntype_t well_known_type;
+  bool is_message_set;
```

Comment:
move the bool up one so that it packs better?

Reply:
Done.